### ISSUE TITLE
Hidden fields container - no padding

### ISSUE THREAD

**Issue by anchovy** [verbb/formie#189]
**What are you trying to do?**
We've got a lot number of hidden fields in our forms, these are all wrapped in the 'row' div's which blow out the height of the forms with whitespace due to the bottom margin:
`.fui-page-row {
    margin: 0 -8px 15px;
}`

![image](https://user-images.githubusercontent.com/599005/96771343-01767380-143e-11eb-955a-d617bec0297f.png)

**What's your proposed solution?**
I can think of a couple of ways to achieve this:
1: Render all hidden fields in a specific div in the form programmatically - ignoring the position specified in the form. If after moving the hidden field the row is empty then don't render the container row.
_Pros_: Quick to implement,
_Cons_: Removes the ability for the user to specify where the hidden fields are rendered in the form.

2: Give the from UI a hidden fields section where the users can place hidden fields knowing this area won't have any padding/margin applied to it.
_Pros_: Allow's users to control where hidden fields are in the form i.e if they really need a hidden field next to another field.
_Cons_: Longer to implement.

**Additional context**

**Comment by engram-design** [comment]
Just added for the next release.

To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.24",
  "...": "..."
}
```

Then run `composer update`.

Ended up introducing `getIsVisible()` property on fields, which will play in nicely for the future when conditional rendering is implemented. When outputting a row, we now check if the row has any visible fields.

I'd rather not introduce a set area for hidden fields, as I've actually come across a few places where I've wanted to do particular things with a field (change it dynamically to another field, or keep it hidden). It's handy knowing wherever you place a field, you know it'll render in that position.

Appreciate the suggestions!

**Comment by engram-design** [comment]
Fixed in [1.2.25](https://github.com/verbb/formie/releases/tag/1.2.25)

**Comment by anchovy** [comment]
Hi @engram-design 

Looks padding has been re-added to the class `.fui-row` and hidden rows are pushing out the display again?


![image](https://user-images.githubusercontent.com/599005/113960489-b24fbe00-9878-11eb-896c-1349f9654cb2.png)

![image](https://user-images.githubusercontent.com/599005/113960342-761c5d80-9878-11eb-8755-7305a8006483.png)

NB - the position of this field is correct, sometimes it's visible, sometimes it's hidden.

Issues:
Craft Pro: 3.5.19.1
Formie: 1.3.25

No issue (i.e no padding  as per dev-craft-3 as 1.2.24)
Craft Pro: Craft Pro 3.5.8
Formie: dev-craft-3

**Comment by engram-design** [comment]
This should be fixed in the upcoming 1.4.0 release, which includes a major refactor on hidden logic to handle conditional fields.
